### ISSUE TITLE
Leptos SSG

### ISSUE THREAD

**Issue by azzamsa** [itehax/itehax_website#3]
Hi.

With the initial SSG feature added to the Leptos https://github.com/leptos-rs/leptos/pull/1649. 
Do you consider using SSG in your blog?

Thank you! 🏅

**Comment by itehax** [comment]
Thanks for the update! I had a look and it looks very nice as a feature,I will wait for it to be completed with the missing features and then I will definitely use it.
I will let you know. 🏅

**Comment by azzamsa** [comment]
Yes. I always wanted to port my blog from 11ty to Rust (Leptos). But I want it to be SSG.

Would you like to elaborate about any missing feature in Leptos SSG?

**Comment by azzamsa** [comment]
Oh, I see. It seems this https://github.com/leptos-rs/leptos/issues/1594#issuecomment-1731985053 is what you mean.

**Comment by itehax** [comment]
Yes exactly! It's just matter of time.
